common/app/NotFound/Not-Found.jsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
-
 // import PropTypes from 'prop-types';
+import Link from '../Router/Link.jsx';
 
 const propTypes = {};
 
 export default function NotFound() {
   return (
-    <div>404 Not Found</div>
+    <div className='container'>
+          <div className='row'>
+            <div className='col-sm'>
+            <div className='alert alert-danger' role='alert'>
+              <h4 className='alert-heading' className='mb-12'>
+                This challenge is not found.</h4>
+                Head back to <Link className='alert-link'
+                   to={'/en/challenges/basic-html-and-html5/' +
+                   'learn-how-freecodecamp-works'}
+                   >
+                    Map
```

Comment:
The Map as it is in production is depreciated, we no longer have a map page, it is just a panel in the view.

The link text should reflect the fact that we are taking them to a challenge